ggshield/scan/scanner.py

```diff
@@ -175,7 +175,13 @@ def __init__(
         ignored_matches: Optional[Iterable[IgnoredMatch]] = None,
         ignored_detectors: Optional[Set[str]] = None,
         ignore_known_secrets: Optional[bool] = None,
+        check_api_health: Optional[bool] = True,
```

Comment:
nitpick: the type for this can be `bool`: it does not need to be optional since it has a default value (making `ignore_known_secrets` optional was a mistake I think)

Reply:
Also, you should set `check_api_health` to False when SecretScanner is created in repo.py.